code/__HELPERS/names.dm

```diff
@@ -235,3 +235,39 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
 				. += "."
 			else
 				. += ", "
+
+/// List of all ship factions to their prefixes.
+GLOBAL_LIST_INIT(ship_faction_to_prefixes, list(
+	"Syndicate" = list(
+		"SEV",
+		"SSV",
+		"SGSV",
+		"SRSV",
+		"STSV",
+	),
+	"Independent" = list(
+		"SV",
+		"IMV",
+		"ISV",
+	),
+	"Inteq Risk Management Group" = list(
+		"IRMV",
+	),
+	"Colonial Minutemen" = list(
+		"CMSV",
+		"CMGSV",
```

Comment:
+CMBSV

Reply:
not in code or jsons anywhere, will not be included.